### DISCUSSION TITLE
Ozone and photolysis schemes

### DISCUSSION THREAD

**Discussion by lukeconibear** [wrfchem-leeds/WRFotron#31] [General]
Hi all.

A while ago, we changed the default settings to use the updated photolysis scheme (`phot_opt = 4`) based on NCAR recommendations. Though, some users are finding that the simulated ozone concentrations aren’t as accurate compared to using the old photolysis scheme (`phot_opt = 3`).

In general, I’ve found the new photolysis scheme simulates lower concentrations of ozone. And depending on the region, this is closer or further away to measurements.

What have others found / think here? There are some settings to change within the new photolysis scheme itself, though I haven’t tested these. (I also found that `phot_opt = 3` would sometimes crash WRFChem4.2 with the mozart_mosaic hanging error.)

**Comment by lukeconibear** [comment]
Benchmarking WRFotron2.3.0 using `phot_opt = 4` found overestimates of O3 in China by 10-30% ([plots](https://github.com/wrfchem-leeds/WRFotron/issues/12#issuecomment-803694331)).

**Comment by clreddington** [comment]
Just including these links for more info (for me):
[https://ruc.noaa.gov/wrf/wrf-chem/wrf_tutorial_2018/photolysis.pdf](url)
[https://www2.acom.ucar.edu/sites/default/files/wrf-chem/Instructions_photolysis_opt_4.pdf](url)

I did spot this, which we could consider:
[https://github.com/wrf-model/WRF/pull/1243](url)
However from a quick look, as the TUV.diags file is written at the start of the run, it doesn't seem to be much of a time drain (maybe ~3mins for me)... Unless I'm missing something and additional diagnostics are written to rsl files during run time
